SU-SWS/stanford_events
# D8CORE-5663 D8CORE-5660 D8CORE-5664 Improve accessibility of the mini calendar

# READY FOR REVIEW

# Summary
- Added screen reader only html for navigation buttons
- Added screen reader only html for current date
- Added aria-live to announce when the view changes.

# Need Review By (Date)
- 4/30

# Urgency
- high.

# Steps to Test
1. checkout this branch
2. add a mini calendar to a page
3. inspect the navigation buttons or use a screen reader
4. verify the accessibility is improved for those.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
